mantidproject/mantid
# Use ParaView to calculate colormap min and max values

Description of work.

This uses the new [FiniteRange member function](https://gitlab.kitware.com/paraview/paraview/merge_requests/1406) to obtain min and max values directly from ParaView and avoids doing the calculation a second time within Mantid.

**To test:**



Load a variety of MDWorkspaces and PeaksWorkspaces in the VSI. Verify that the colormap still behaves as expected.

There is no GitHub issue associated with this pull request.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
